lib/model/rwfolder.go

```diff
@@ -254,7 +230,9 @@ func (f *sendReceiveFolder) Serve() {
 						})
 					}
 
-					f.pullTimer.Reset(f.pause)
+					time.AfterFunc(f.pause, f.SchedulePull)
+					f.increasePause()
```

Comment:
Given this function is only called once, I'd put it's code here for clarity and to remove indirection.